lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -96,7 +96,7 @@
                                 </mat-icon>
                                 <ng-template #no_iconvalue>
                                     <mat-icon class="adf-datatable-selected"
-                                              *ngIf="row.isSelected; else no_selected_row" svgIcon="selected">
+                                              *ngIf="row.isSelected && col.key=='$thumbnail'; else no_selected_row" svgIcon="selected">
```

Comment:
What is the functionality that is achieved with this change?

Reply:
The change is that only columns with '$thumbnail' as key (the first one) is replaced with the selected icon, not all mat-icon columns.